lib/Board/board.js

```diff
@@ -6,31 +6,77 @@ export class Board {
         this.cells = new Array(this.rowTotal).fill('').map(() => new Array(this.columnTotal).fill(''));
     };
 
-    /* getCells():Array */
+    // public toString(): String 
+    toString() {
+        return this.#assembleBoard();
+    };
+
+    // public getCells(): Array 
     getCells() {
         return this.cells;
     };
 
-    /* toString():String */
-    toString() {
-        return this.assembleBoard();
+    // public getRow(): Number
+    getRow(position) {
+        return this.#findRow(position);
+    };
+
+    // public getColumn(): Number
+    getColumn(position) {
```

Comment:
The type for `getColumn()` should take an argument! What's the argument's data type?